apps/nextjs-website/src/components/atoms/ChatbotHistoryNavigationLink/ChatbotHistoryNavigationLink.tsx

```diff
@@ -0,0 +1,36 @@
+import { Link, useTheme } from '@mui/material';
+
+type ChatbotHistoryNavigationLinkProps = {
+  sessionId: string;
+  sessionTitle: string;
+};
+
+const ChatbotHistoryNavigationLink = ({
+  sessionId,
+  sessionTitle,
+}: ChatbotHistoryNavigationLinkProps) => {
+  const { palette, typography } = useTheme();
+  const textColor = palette.text.secondary;
+
+  return (
+    <Link
+      maxWidth='15rem'
+      textOverflow='ellipsis'
+      color={textColor}
+      fontSize='1.125rem'
+      fontWeight='600'
+      fontFamily={typography.fontFamily}
+      component='span'
+      noWrap
+      sx={{ cursor: 'pointer', textDecoration: 'none' }}
+      onClick={() => {
+        // TODO: Implement the navigation to the chatbot history session
+        console.log(`Navigating to chatbot history session: ${sessionId}`);
+      }}
+    >
```

Comment:
Maybe console log can be removed moving the responsibilities of changing page out of this atom and exposing a function to handle the event. 
NB if you applay yhis suggestion remember to update other component accordly :wink: 
```suggestion
type ChatbotHistoryNavigationLinkProps = {
  sessionTitle: string;
  onSessionChange?: () => null;
};

const ChatbotHistoryNavigationLink = ({
  sessionTitle,
  onSessionChange,
}: ChatbotHistoryNavigationLinkProps) => {
  const { palette, typography } = useTheme();
  const textColor = palette.text.secondary;

  return (
    <Link
      maxWidth='15rem'
      textOverflow='ellipsis'
      color={textColor}
      fontSize='1.125rem'
      fontWeight='600'
      fontFamily={typography.fontFamily}
      component='span'
      noWrap
      sx={{ cursor: 'pointer', textDecoration: 'none' }}
      onClick={onSessionChange}
    >
```